parser_library/src/expressions/mach_expr_term.h

```diff
@@ -16,6 +16,7 @@
 #define HLASMPLUGIN_PARSERLIBRARY_EXPRESSIONS_MACH_EXPR_TERM_H
 
 #include "context/id_storage.h"
+#include "context/ordinary_assembly/symbol_value.h"
```

Comment:
Maybe forward declaration can be used here or in `mach_expression.h` as well and we can get rid off the header by doing so?

But I'm not sure to what extent you want to go with the dependency reduction

Reply:
That's probably just because of `value_t` which could be moved here. But I don't really mind if it stays as it is.

Reply:
value_t == symbol_value